target/linux/ath79/dts/qca9563_dlink_dir-842-c2.dts

```diff
@@ -4,23 +4,12 @@
 #include <dt-bindings/gpio/gpio.h>
 #include <dt-bindings/input/input.h>
```

Comment:
These includes are already in the dtsi...

Reply:
Removed, thanks!